server/views/components/series-container-async/promos-list.njk

```diff
@@ -0,0 +1,27 @@
+{% for promo in promos %}
+  {% if loop.length % 3 == 0 %}
```

Comment:
There's a bit of inconsistency in the equality operator going on – `===` all the way?